cli/azd/pkg/tools/azcli/webapp.go

```diff
@@ -51,7 +51,8 @@ func (cli *azCli) appService(
 
 func isLinuxWebApp(response *armappservice.WebAppsClientGetResponse) bool {
 	if response.Properties != nil && response.Properties.SiteConfig != nil &&
-		response.Properties.SiteConfig.LinuxFxVersion != nil {
+		response.Properties.SiteConfig.LinuxFxVersion != nil &&
```

Comment:
do we want test cases for this new condition (add a test with a mock response returning empty string for LinuxFxVersion